# j1: joint 1 angle (radians)
# j2: joint 2 angle (radians)
float64 j1 
float64 j2 
